justfile

```
@@ -0,0 +1,19 @@
+
+test *args: (test-forge args)
+build *args: (build-forge args)
+prep *args: fix (test args)
+
+
+test-forge *args: install-forge build-forge
+    forge test --isolate --no-match-path 'test/js-scripts/**/*' {{args}}
```

not exactly sure how nested commands work with just, but looked like we were calling `forge install` twice if you can `just test`

```suggestion
prep *args: fix (test-forge args)


test-forge *args: build-forge
    forge test --isolate --no-match-path 'test/js-scripts/**/*' {{args}}
```

---

yeah ur prob right good catch